src/librustc_mir/borrow_check/mod.rs

```diff
@@ -2179,17 +2242,21 @@ impl<'cx, 'tcx> MirBorrowckCtxt<'cx, 'tcx> {
                     | ProjectionElem::ConstantIndex { .. }
                     | ProjectionElem::Subslice { .. }
                     | ProjectionElem::Downcast(..) => {
-                        let upvar_field_projection = self.is_upvar_field_projection(place);
+                        let upvar_field_projection = self.is_upvar_field_projection(&Place {
+                            base: place_base.clone(),
+                            projection: place_projection.clone(),
+                        });
                         if let Some(field) = upvar_field_projection {
                             let upvar = &self.upvars[field.index()];
                             debug!(
-                                "upvar.mutability={:?} local_mutation_is_allowed={:?} place={:?}",
-                                upvar, is_local_mutation_allowed, place
+                                "upvar.mutability={:?} local_mutation_is_allowed={:?}" /
```

Comment:
```suggestion
                                "upvar.mutability={:?} local_mutation_is_allowed={:?} \
```

Reply:
Hehe I was sleepy yesterday trying to push something quickly and wrote that in another language :P.